ui/client/components/graph/NodeUtils.ts

```diff
@@ -15,16 +26,16 @@ class NodeUtils {
     return node.type ? node.type : "Properties"
   }
 
-  nodeIsProperties = (node: NodeType): boolean => {
+  nodeIsProperties = (node: NodeType | GroupType): node is PropertiesType => {
```

Comment:
this types are still temporary. I have to avoid errors by adding this. it's really hard to identify this types, to do this partially, especially by looking on "utils" like this - js functions (with loadsh, strings etc.) accepting everything, returning anything ;)